### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by morfeus2** [popcornmix/omxplayer#543]
Hi,
omxplayer is very good piece of software, essential for enjoying video on raspberry, but I am missing some of the features of Mplayer.
I managed to overcome the lack for lirc interface, by using dbus-send and irexec.
First thing I can't solve: if I understand correctly, only delay for subtitles is implemented, while delay for A-V sync is not there.
Would it be possible to implement it?

Second question: since raspberry has no audio card, any suggestion for a way to send audio from omxplayer running on raspberry, through ethernet to a x86 PC (Debian), that has a good audio equipment connected ?

**Comment by popcornmix** [comment]
Recent omxplayer version support alsa devices (e.g. USB and I2S audio cards).

**Comment by morfeus2** [comment]
Yes, I noticed, and that is nice; but I have my audio system "far" from the raspberry (that is sitting below a projector), so instead of routing an audio cable it would be more convenient to stream audio directly through ethernet.
Apart from this (that maybe I will manage by adding a USB->audio and routigh analog audio through the unused twisted pairs of the ethernet cable) what about the other question for adjusting audio/video sync/delay? I find a bit strange that it was not implemented.

**Comment by popcornmix** [comment]
Yes, looks like A/V offset has never been plumbed in.
omxplayer does handle it internally.
It would need a call to:
```m_player_video.SetDelay(new_delay)```
similar to the calls to:
```m_player_subtitles.SetDelay(new_delay)```

**Comment by morfeus2** [comment]
Exactly! If only I was good enough to code such an addition...
Unfortunately my skills in C are definitely lower than the necessary.
